src/Host/Broker/Impl/Lifetime/PingController.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Microsoft.R.Host.Broker.Lifetime {
```

Comment:
This isn't actually used by VS side at the moment, and lifetime is managed by PID watching instead. I kept it around in case we need it later.